Resources/doc/reference/configuration/invalidation.rst

```diff
@@ -20,7 +19,25 @@ routes when some other routes are requested.
                         villain_details:     # e.g., /villain/{id}
                             ignore_extra_params: false    # Defaults to true
 
-.. include:: /includes/enabled.rst
+.. include:: ../../includes/enabled.rst
```

Comment:
i am running sphinx-build -v 1.2.2 so maybe you would need to upgrade.
did you run from inside the doc/ directory and type `make html`?
otherwise sphinx will not be configured correctly